frappe/utils/file_lock.py

```diff
@@ -41,6 +42,15 @@ def lock_exists(name):
 	return os.path.exists(get_lock_path(name))
 
 
+def lock_age(name) -> float:
+	"""Return time in seconds since lock was created."""
+	mtime = Path(get_lock_path(name)).stat().st_mtime
+	if not mtime:
+		return None
+
+	return time() - mtime
+
+
```

Comment:
Can maybe simplify this

```suggestion
def lock_age(name) -> float | None:
	"""Return time in seconds since lock was created."""
	if mtime := Path(get_lock_path(name)).stat().st_mtime:
		return time() - mtime
	return None
	
```